### PR TITLE
fix: <template> tag for browsers that do not support them

### PR DESCRIPTION
**Pull request by yjbanov**

### CODE REVIEW

File: modules/angular2/src/dom/browser_adapter.dart [hunk 1]
```diff
@@ -207,7 +215,7 @@ class BrowserDomAdapter extends GenericBrowserDomAdapter {
   String nodeName(Node el) => el.nodeName;
   String nodeValue(Node el) => el.nodeValue;
   String type(InputElement el) => el.type;
-  Node content(TemplateElement el) => el.content;
+  Node content(TemplateElement el) => _supportsTemplateElement ? el.content : el;
   Node firstChild(el) => el.firstChild;
```
Comment (caitp) [inline modules/angular2/src/dom/browser_adapter.dart]: The supported flag should probably be memoized

Comment (caitp) [inline modules/angular2/src/dom/browser_adapter.dart]: Misread, guess it is

**Comment by vicb** [comment]
I guess `templateAwareRoot()`  also needs to be modified ?

**Comment by mhevery** [comment]
@vicb `templateAwareRoot` already has the check. this should be LGTM

**Comment by vicb** [comment]
@mhevery I was not sure if `is TemplateElement` would work for those browser

**Pull request closed without merging**